snapshots/testsuite/helpers.go

```diff
@@ -140,28 +140,28 @@ func checkSnapshots(ctx context.Context, sn snapshots.Snapshotter, work string,
 // checkInfo checks that the infos are the same
 func checkInfo(si1, si2 snapshots.Info) error {
 	if si1.Kind != si2.Kind {
-		return fmt.Errorf("Expected kind %v, got %v", si1.Kind, si2.Kind)
+		return fmt.Errorf("expected kind %v, got %v", si1.Kind, si2.Kind)
 	}
 	if si1.Name != si2.Name {
-		return fmt.Errorf("Expected name %v, got %v", si1.Name, si2.Name)
+		return fmt.Errorf("expected name %v, got %v", si1.Name, si2.Name)
 	}
 	if si1.Parent != si2.Parent {
-		return fmt.Errorf("Expected Parent %v, got %v", si1.Parent, si2.Parent)
+		return fmt.Errorf("expected Parent %v, got %v", si1.Parent, si2.Parent)
```

Comment:
Would you mind to show the fmt error you try to fix? Thanks

Reply:
Error string should not be capitalized or end with punctuation mark